apps/src/StudioApp.js

```diff
@@ -1212,7 +1214,7 @@ StudioApp.prototype.showInstructionsDialog_ = function (level, autoClose, showHi
 
   var hideFn = _.bind(function () {
     // Momentarily flash the instruction block white then back to regular.
-    if ($(endTargetSelector).length) {
+    if ($(endTargetSelector).length && !instructionsInTopPane) {
```

Comment:
Nit: It will be _slightly_ faster to do the boolean check before the DOM lookup, in the `instructionsInTopPane` case.